.taskcluster.yml

```diff
@@ -15,25 +15,25 @@ tasks:
               $if: 'tasks_for in ["github-push", "github-release"]'
               then: '${event.repository.html_url}'
               else:
-                  $if: 'tasks_for == "github-pull-request"'
+                  $if: 'tasks_for[:19] == "github-pull-request"'
```

Comment:
do you remember why we need to do the string slice here? I don't think github service will send any strange values for the `tasks_for`

Reply:
I was just doing similar to taskgraph https://github.com/taskcluster/taskgraph/blob/main/.taskcluster.yml

Reply:
aah. this is probably because of the new value that Ahal added - `github-pull-request-unrestricted` .. ok

Reply:
Yeah, I think you're right. That makes more sense